default/src/components/asaTransfer.js

```diff
@@ -0,0 +1,69 @@
+/* global AlgoSigner */
+import algosdk from 'algosdk';
+
+import { networks } from '../../algob.config';
+import { LEDGER } from '../algosigner.config';
+
+const LAST_ROUND = 'last-round';
+const CONFIRMED_ROUND = 'confirmed-round';
+
+const algodClient = new algosdk.Algodv2(
+  networks.default.token,
+  networks.default.host,
+  networks.default.port
+);
+
+const waitForConfirmation = async function (txId) {
+  let response = await algodClient.status().do();
+  let lastround = response[LAST_ROUND];
+  while (true) {
+    const pendingInfo = await algodClient
+      .pendingTransactionInformation(txId)
+      .do();
+    if (
+      pendingInfo[CONFIRMED_ROUND] !== null &&
+      pendingInfo[CONFIRMED_ROUND] > 0
+    ) {
+      //Got the completed Transaction
+      return (
+        'Transaction ' +
+        txId +
+        ' confirmed in round ' +
+        pendingInfo[CONFIRMED_ROUND]
+      );
+    }
+    lastround++;
+    await algodClient.statusAfterBlock(lastround).do();
+  }
+};
+
+async function ASATransfer(asaId, sndrAddr, recvAddr, amount) {
```

Comment:
```suggestion
async function transferASA(asaId, sndrAddr, recvAddr, amount) {
```

Reply:
Also we shouldn't start function/variables declarations with capital letter. They are used for declaring types, interfaces, constant values (eg. `MAX_UINT64`, `TransactionType`..)